dev/io.openliberty.grpc.client.1.0.thirdparty/bnd.bnd

```diff
@@ -16,12 +16,13 @@
 
```

Comment:
Update copyright date range to include 2023

Reply:
Done